### PR TITLE
Update plugins docs for Jib 3.0

### PR DESCRIPTION
**Pull request by chanseokoh**
Is there anything else we missed?

### CODE REVIEW

File: jib-gradle-plugin/README.md [hunk 1]
```diff
@@ -426,6 +429,19 @@ You may also specify the target of the copy using `paths` as a closure:
         from = file('src/main/another/dir')
         into = '/extras'
       }
+      path {
+        // copies a single file
```
Comment (mpeddada1) [inline jib-gradle-plugin/README.md:433]:
```suggestion
        // copies a single-file.xml
```
```suggestion
        // copies a single file
```

File: jib-maven-plugin/README.md [hunk 2]
```diff
@@ -309,6 +309,8 @@ Property | Type | Default | Description
 --- | --- | --- | ---
 `from` | file | `[(project-dir)/src/main/jib]` | The source directory. Can be absolute or relative to the project root.
 `into` | string | `/` | The absolute unix path on the container to copy the extra directory contents into.
+`includes` | list | *None* | Glob patterns for including files. See [Adding Arbitrary Files to the Image](#adding-arbitrary-files-to-the-image) for an example.
```
Comment (mpeddada1) [inline jib-maven-plugin/README.md:312]: Do we want to document that glob pattern matching here behaves differently from maven-resources-plugin?

Comment (chanseokoh) [reply jib-maven-plugin/README.md:312]: I think it's okay without it. People will follow the link to see the example, where I added the correct usage to cover the case we discussed.

File: jib-gradle-plugin/README.md [hunk 3]
```diff
@@ -193,6 +193,7 @@ Field | Type | Default | Description
...
 `containerizingMode` | `String` | `exploded` | If set to `packaged`, puts the JAR artifact built by the Gradle Java plugin into the final image. If set to `exploded` (default), containerizes individual `.class` files and resources files.
 `allowInsecureRegistries` | `boolean` | `false` | If set to true, Jib ignores HTTPS certificate errors and may fall back to HTTP as a last resort. Leaving this parameter set to `false` is strongly recommended, since HTTP communication is unencrypted and visible to others on the network, and insecure HTTPS is no better than plain HTTP. [If accessing a registry with a self-signed certificate, adding the certificate to your Java runtime's trusted keys](https://github.com/GoogleContainerTools/jib/tree/master/docs/self_sign_cert.md) may be an alternative to enabling this option.
+`configurationName` | `String` | `main` | Specify the name of the [Gradle Configuration](https://docs.gradle.org/current/dsl/org.gradle.api.artifacts.ConfigurationContainer.html) to use.
```
Comment (mpeddada1) [inline jib-gradle-plugin/README.md:196]: quick question: I thought the default configuration name was `runtimeClasspath`?

Comment (chanseokoh) [reply jib-gradle-plugin/README.md:196]: Oh, you're right! Nice catch.

**Review by mpeddada1 — APPROVED** [review]

**Pull request merged**